eclipse-scout-core/src/form/fields/wrappedform/WrappedFormField.ts

```diff
@@ -65,21 +74,16 @@ export class WrappedFormField extends FormField implements WrappedFormFieldModel
       return;
     }
 
-    this.innerForm.setDisplayHint(Form.DisplayHint.VIEW); // by definition, an inner form is a view.
-    this.innerForm.setModal(false); // by definition, an inner form is not modal.
-    this.innerForm.setClosable(false); // Disable close key stroke
-    this.innerForm.renderInitialFocusEnabled = this.initialFocusEnabled; // do not render initial focus of form if disabled.
-    if (this.innerForm.rootGroupBox) {
-      this.innerForm.rootGroupBox.setMenuBarPosition(GroupBox.MenuBarPosition.BOTTOM);
-    }
-
+    this.innerForm.renderInitialFocusEnabled = this.initialFocusEnabled;
```

Comment:
This modifies the model of the innerForm. I think we should move this call to `_setInnerForm` (or `_adaptForm`) and `_setInitialFocusEnabled`